src/components/Exercise/index.tsx

```diff
@@ -195,15 +213,32 @@ export const Exercise = styled(({
       {...(exerciseIcons ? { exerciseIcons: exerciseIcons } : null)}
       className={props.className}
     >
-      <div ref={container}>
+      <div
+        ref={hoverRef}
+        tabIndex={enableOverlay ? 0 : -1} // This container is focusable only if enableOverlay is true
```

Comment:
Does this cause any side effects for non-overlay exercises in screenreaders? I'm wondering if we should be more conservative and not set the attribute at all for normal use.